hw03_mohamedm.py

```diff
@@ -0,0 +1,117 @@
+#################################################################################
+# Author: Malak Mohamed
+# Username: mohamedm
+#
+# Assignment: HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose: exploring turtles
+# Google Doc Link: https://docs.google.com/document/d/1y-CIy7MdTK91WeMf1-7qtPrdj9Rpr0n_j_vf17A3Qe8/edit?usp=sharing
+#
+# Acknowledgements: https://stackoverflow.com/questions/29441237/how-to-draw-a-semicircle-in-python-turtle-only  used for doing the pattern function
+#
+#################################################################################
+
+import turtle
+
```

Comment:
Pay attention to gray squigglies, they indicate style guidelines. This one asks for two blank lines before all function definitions